### DISCUSSION TITLE
SFTP sudo User

### DISCUSSION THREAD

**Discussion by Sh4d0w0561** [isontheline/pro.webssh.net#1010] [Ideas]
I miss the sudo function in SFTP, which allows a user to open the directories + files with sudo. This works in WinSCP. Can this also be added in the app?

**Comment by isontheline** [comment]
I need to check if I can implement this as I wasn't aware.

How do you achieve the same behavior within Filezilla (instead of WinSCP)?

**Comment by Sh4d0w0561** [comment]
sudo /usr/lib/openssh/sftp-server
https://winscp.net/eng/docs/faq_su

**Reply by isontheline** [reply]
Right now I'm unable to pass a custom subsystem command. I need to find how.

**Reply by Sh4d0w0561** [reply]
any new news? 🙂

**Reply by isontheline** [reply]
My current implementation doesn't allow me to pass a custom subsystem binary path.

Why not adding a user with proper privileges?

**Reply by Sh4d0w0561** [reply]
Allready done, i have disabled root login for security reasons. The new user are already in root group, but without sudo i can't edit any file. That's the reason, why i have created this thread. With WinSCP work sudo fine.

**Reply by isontheline** [reply]
I'm opening an issue to track your request : https://github.com/isontheline/pro.webssh.net/issues/1017
